Lib/test/_test_multiprocessing.py

```diff
@@ -5022,6 +5022,8 @@ def test_get_all(self):
         methods = multiprocessing.get_all_start_methods()
         if sys.platform == 'win32':
             self.assertEqual(methods, ['spawn'])
+        elif sys.platform == 'darwin':
+            self.assertEqual(methods, ['spawn', 'fork', 'forkserver'])
```

Comment:
ISTM this should allow 'forkserver' to be missing, as done by the assertion two lines below.

Reply:
Droped this test, I agree it's redundant + fixed the code to have the return value in the else clause